### PR TITLE
Packaging and README

### LINKED ISSUE #1: Update readme file
- point people to topography file website
- describe track input file
- description of what code does
- how to install it and use it
- things people need/can to change (ex. path to input files, resolution, etc.)
- how people can contribute

### PR DESCRIPTION
**Pull request by jatkinson1000**
Closes #1 

Incorporates PR #3 

I have added a pyproject.toml file for packaging up the repo and easy installation, and added instructions to the README.

Laura has updated the readme with key info.

### CODE REVIEW

**Comment by lcimoli** [comment]
Added details to README file. 
@jatkinson1000 Both this branch and the readme one had an updated README, so I chose to put all the changes in here. Once this is closed, the readme branch should also be removed (I guess?). Also, I could not add you as a reviewer because you created the pull reques (right?), so I've added you as an assignee - hope that's okay.

**Comment by jatkinson1000** [comment]
@lcimoli Yes, that's right, the person who opened the PR can't 'approve'. Not to worry - its a learning process.
I can still review others' code, so will do that shortly for you.
I updated the description to say that we are incorporating #3 , so yes, you can close that (perhaps with a note to say it has been incorporated here).

File: README.md [hunk 1]
```diff
@@ -1,2 +1,42 @@
-# station_bathymetry
-Plot lat/lon bathymetry at station
+## Station bathymetry
+The purpose of the script is to check the bathymetry at a station in preparation of a CTD cast. The main concern is that the CTD might drift upgradient, into the topography, and hit the bottom. Knowing the bathymetry a priori helps avoiding such issue, and can help 
+- informing the bridge about what direction to drift (the resolution of the bathymetry used here is often higher than the one of the nautical charts available onboard);
+- deciding whether to stop the cast slightly before (e.g. 15m from the bottom);
+- deciding whether to ask the whinch to recover the package faster than usual (e.g. 75 mpm rather than the standard 60 mpm);
```
Comment (jatkinson1000) [inline README.md]: Spelling: whinch --> winch

File: README.md [hunk 2]
```diff
@@ -1,2 +1,42 @@
...
+	python3 -m pip install .
+
+It is recommended to use a virtual environment.
+
+### Getting started
+
+The plotting tool takes the form of a jupyter notebook `plot_bathymetry_at_station.ipynb`.
+This can be run using:
+
+	pip install notebook
```
Comment (jatkinson1000) [inline README.md]:
```suggestion
```

No need to pip install notebook, as it should be covered in the above `pip` statement.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,42 @@
...
+   	jupyter notebook plot_bathymetry_at_station.ipynb
+   	
+
+### Code Description
+Please check the jupyter notebook for details about the code steps, input files and user-controlled parameters.
+ 
+
+
+### Contributions
+Contributions and collaborations are welcomed from anyone with an interest in [GO-SHIP](https://www.go-ship.org/), and more in general ship-based hydrographic observations and python.
```
Comment (jatkinson1000) [inline README.md:37]:
```suggestion
Contributions and collaborations are welcomed from anyone with an interest in [GO-SHIP](https://www.go-ship.org/), and more general ship-based hydrographic observations and python.
```

**Review by jatkinson1000 — COMMENTED** [review]
In general this looks pretty good to me.
I have made a couple of suggestions and typo changes.

We could also consider adding a badge to the top of the readme with the license info using
```
![GitHub](https://img.shields.io/github/license/GO-SHIP-Oceanography/station_bathymetry)
```
(other badges are also available e.g. python version).

File: README.md (hunk 2 above)
Comment (lcimoli) [reply README.md]: Fixed

**Review by lcimoli — APPROVED** [review]

**Pull request merged**